game-core/src/main/java/games/strategy/triplea/delegate/EndTurnDelegate.java

```diff
@@ -216,12 +217,28 @@ private String addUnitCreatedResources(final IDelegateBridge bridge) {
   }
 
   /**
-   * Find the PU value of national objectives that are currently met.
+   * Find the resources generated by national objectives and triggers that are currently met.
    */
-  public static int findNationalObjectivePus(final PlayerID player, final GameData data, final IDelegateBridge bridge) {
+  public static IntegerMap<Resource> findNationalObjectiveAndTriggerResources(final PlayerID player,
```

Comment:
This method appears to only be called from `AbstractEndTurnDelegate`.  Can it be moved there and made private, or do you have plans for it in a forthcoming PR?  (Moving it would also get rid of one more instance of that smell in `AbstractEndTurnDelegate` where a superclass calls a static method in a subclass.)

Reply:
Yeah, I'm leaving it there for now as its similar to the method below it and want to further refactor them.